src/main/java/socket/TFTPDatagramSocket.java

```diff
@@ -42,7 +43,7 @@ public DatagramPacket receiveMessage() throws IOException
 		DatagramPacket packet = new DatagramPacket(new byte [Message.MAX_PACKET_SIZE], Message.MAX_PACKET_SIZE);
 		receive(packet);
 		LOG.logVerbose("Received data from " + packet.getSocketAddress());
-		LOG.logVerbose(packet.getData());
+		LOG.logVerbose(packet);
```

Comment:
AFAICT this is the only place where we call the logger with the byte array.

Reply:
Yes you're absolutely right, I happened to use byte logging lot when adding features in or testing  new changes out, but eventually the log statements get removed as they are more or less for my own debugging purposes. 

It was more of a utility to help with development

Reply:
I find it really useful for debugging as well, I think we should keep it.